README.md

```diff
@@ -18,6 +18,10 @@ exceptions/differences/extensions (:white_check_mark: are the implemented sniffs
 
 - Keep the nesting of control structures per method as small as possible
 - Prefer early exit over nesting conditions or using else
+- :white_check_mark: Abstract classes should not be prefixed with `Abstract`
+- :white_check_mark: Interfaces should not be suffixed with `Interface`
+- :white_check_mark: Concrete exception class names should not be suffixed with `Exception`
+- :white_check_mark: Abstract exception class names and exception interface names should be suffixed with `Exception`
```

Comment:
As @kukulich said, this isn’t possible 😞

Reply:
Why not? You can assume inheritance - when ancestor has the `Exception` suffix, this class has too.

Reply:
Yes, you're right but:

- Not all exceptions has `Exception` suffix, eg. `TypeError`.
- You cannot report all bugs in one run with code like `A extends B extends C extends Exception`. You can report `C` in first run and `B` in second.

So if you're ok with this, it's possible to check "Abstract exception class names". However, I think it's really not possible to report "exception interface names should be suffixed with `Exception`"

Reply:
I'll remove the check mark on that one then.

Reply:
Argh @Ocramius just merged :P

Reply:
@greg0ire fixup patch with discussion there please :P

Reply:
okay